server/fly.io/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "cargo-quickinstall-stats-server"
+version = "0.0.0"
+authors = ["David Laban <alsuren@gmail.com>"]
+# based on https://github.com/fly-apps/hello-rust by Jerome Gravel-Niquet <jeromegn@gmail.com>
+edition = "2021"
+
+# Make a new workspace so that we get our own Cargo.lock and target dir for the docker build. 
+[workspace]
+
+[dependencies]
+tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
+axum = "0.6.10"
+influx_db_client = "^0.5.0"
+influxrs = "2.0.1"
```

Comment:
placeholder to think about https://github.com/ijagberg/influx/issues/4 at some point. This might involve forking their repo temporarily and deploying from the fork

Reply:
oh you know what? it might be a problem with how the js server is encoding the url params?

(still not looked into it properly though)

Reply:
Fixed in https://github.com/ijagberg/influx/issues/5 . Depending on my fork for now.